PotteryBarn/CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
 ## Changelog
+### 1.17.0
```

Comment:
Empty newline after changelog, change this to version `1.18.0` and period to end the sentence.